content_sync/pipelines/definitions/concourse/site-pipeline.yml

```diff
@@ -383,15 +383,15 @@ jobs:
                 mkdir -p ./static/static_shared
                 if [ ! -z "$(ls -A ../static-resources)" ];
                 then
-                  find ../static-resources ! -name '*.mp4' ! -name '*.js.map' -type f | xargs cp -t ./content/static_resources
+                  find ../static-resources ! -name '*.mp4' -type f | xargs cp -t ./content/static_resources
                 fi
                 HTML_COUNT="$(ls -1 ./content/static_resources/*.html 2>/dev/null | wc -l)"
                 if [ $HTML_COUNT != 0 ];
                 then
                   mv ./content/static_resources/*.html ./static/static_resources
                 fi
                 touch ./content/static_resources/_index.md
-                aws s3((cli-endpoint-url)) sync s3://((web-bucket))/static_shared ./static/static_shared
+                aws s3((cli-endpoint-url)) sync s3://((web-bucket))/static_shared --exclude *.js.map ./static/static_shared
```

Comment:
This `--exclude` argument needs to go at the end of the command. The `aws s3 sync` command expects 2 positional arguments after `sync`, the source and the destination. You can't place keyword arguments in the middle like this, as it produces an error.